packages/hopp-plugin-concat/lib/index.js

```diff
@@ -18,7 +18,35 @@ export const config = {
 
 /**
  * We don't need to do any real transformation.
+ * 
+ * suffix and prefix features
+ * @param suffix || null @param prefix || null
+ * appending additional suffix and prefix to each data.
  */
 export default async (ctx, data) => {
+  /**
+   * unexpected num of arguments handling
+   */
+  if (ctx.args.length > 1) {
+    throw new Error('Unexpected numbers of arguments.')
+  }
+
+  /**
+   * options assign
+   */
+  var opts = Object.assign({}, ctx.args[0] || {})
```

Comment:
This is only necessary if you are going to modify the opts object, which you aren't. Since you are not modifying the opts, you can just do this: `const opts = ctx.args[0]`

Also avoid using `var` in ES2015 code. Use `let` or `const`.